packages/@dcl/inspector/src/components/ui/EntityField/EntityField.tsx

```diff
@@ -1,25 +1,26 @@
-import React, { useMemo } from 'react'
+import React, { useMemo, useState } from 'react'
 import cx from 'classnames'
 import { BiCube as EntityIcon } from 'react-icons/bi'
 import { Entity } from '@dcl/ecs'
 import { withSdk, WithSdkProps } from '../../../hoc/withSdk'
 import { Dropdown } from '../Dropdown'
 import type { Props } from './types'
+import { useSelectedEntity } from '../../../hooks/sdk/useSelectedEntity'
```

Comment:
could you please move this import below the `import { withSdk, WithSdkProps } from '../../../hoc/withSdk'`?